library/spdm_crypt_lib/libspdm_crypt_asym.c

```diff
@@ -757,24 +911,55 @@ bool libspdm_asym_verify(
         if (!result) {
             return false;
         }
-        return libspdm_asym_verify_wrap(context, hash_nid, base_asym_algo,
-                                        param, param_size,
-                                        message_hash, hash_size,
-                                        signature, sig_size);
+        result = false;
```

Comment:
I put the 
```
if (try_big_endian) {....}
if (result == false && try_little_endian) {...}
```
endian handling pattern here at this level since the spdm_version check is here is available here. 
Looking at this again, we could put this endian handling inside the `libspdm_asym_verify_wrap` function instead, to save some code and potential for missing it, but that would not have a spdm_version check at the same layer. I'm open to which ever what you prefer.